src/main/resources/application.yaml

```diff
@@ -27,6 +27,7 @@ spring:
 
   data:
     web:
+      allowedOriginsUrls: ""
```

Comment:
the allowedOrigins should not be placed in the spring.web.data section, but rather under server one.